array_equals.rb

```diff
@@ -0,0 +1,21 @@
+# Determines if the two input arrays have the same count of elements
+# and the same integer values in the same exact order
+def array_equals(array1, array2)
+  if array1 == nil && array2 == nil
+    return true
+  elsif array1 == nil && array2 != nil
+    return false
+  elsif array1 != nil && array2 == nil
```

Comment:
Lines 6 and 8 seem really similar.  Do you think there is a way you could combine them to make it more DRY?

Reply:
Oh, that makes sense I went back and combine them and they work. Thanks :)
The refactored code is below:
`elsif (array1 == nil && array2 != nil) || (array1 != nil && array2 == nil)
    return false`